stock_warehouse_flow_delivery_refresh/models/stock_move.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2025 Camptocamp SA, BCIM
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _stock_warehouse_flow_onchange_carrier(self):
```

Comment:
I think naming without using `onchange_carrier` is more accurate, because this method doesn't know where it was called from, therefor it is enough to name it after what i actual does. 
```suggestion
    def _refresh_warehouse_flow(self):
```
or
```suggestion
    def _update_warehouse_flow(self):
```